common/libs/wallet/IWallet.ts

```diff
@@ -1,6 +1,6 @@
 import { RawTransaction } from 'libs/transaction';
 
 export interface IWallet {
-  getAddress(): Promise<string>;
-  signRawTransaction(tx: RawTransaction): Promise<string>;
+  getAddressString(): Promise<string> | string;
```

Comment:
Any reason for the name change? Is address ever expected to be a non-string?

Also, how is the ambiguity of promise or string handled?

Reply:
Other than reducing the amount of rewriting in one pr, is there any reason we need to keep these as promiseable anyway?

Reply:
getAddress (in the ethereumjs-wallet package API) returns a buffer in ethereumjs-wallet so you need to use the string post-fixed version)

Reply:
We keep them as promisifiable because deterministic wallets are actually async vs the non-deterministic ones